### PR TITLE
AGENT-153: Promote assisted-installer to OCP namespace

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
Promote assisted-installer to OCP namespace

Ref: https://issues.redhat.com/browse/AGENT-153

/cc @lranjbar @andfasano

### CODE REVIEW

File: ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml [hunk 1]
```diff
@@ -23,17 +23,24 @@ base_images:
...
+  from: base
   inputs:
-    openshift_release_golang-1.17:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
-      - registry.ci.openshift.org/openshift/release:golang-1.17
-  to: assisted-installer
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
+  to: agent-installer-orchestrator
 - dockerfile_path: Dockerfile.assisted-installer-controller
```
Comment (pawanpinjarkar) [inline ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml:44]: Maybe this image is not needed here as well as we have the `edge` variant.

Comment (pawanpinjarkar) [reply ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml:44]: Nit: `dockerfile_path: Dockerfile.assisted-installer-controller`

Comment (lranjbar) [reply ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml:44]: The assisted-installer-controller needs to stick around. I know its confusing two containers come out of the assisted-installer repo.

The approved OCP payload name is `agent-installer-csr-approver` and file is `Dockerfile.assisted-installer-controller.ocp`

**Review by lranjbar — CHANGES_REQUESTED** [review]
Just clearing up this tidbit.

**Comment by andfasano** [comment]
@pawanpinjarkar looks like you need to run a `make ci-operator-config`

**Pull request merged**